R/01-webdata-obj.R

```diff
@@ -114,12 +114,16 @@ setMethod("webdata", signature("character"), function(.Object=c("prism",  "iclus
 #' @aliases webdata
 setMethod("webdata", signature("geojob"), function(.Object, ...) {
   xmlVals <- inputs(xmlParse(xml(.Object)))
+  browser()
   url <- xmlVals[["DATASET_URI"]]
   times <- c(start = xmlVals[["TIME_START"]], end = xmlVals[["TIME_END"]])
   if(is.null(times[['start']])) {times[['start']] <- NA}
   if(length(times) == 1) {times[['end']] <- NA}
-  times <- as.POSIXct(c(times[['start']], times[['end']])) #could get out of order with one missing
-
+  times <- c(times[['start']], times[['end']])#could get out of order with one missing
+  times <- gsub(pattern = "T", replacement = " ", x = times)
+  times <- as.POSIXct(strftime(times, 
```

Comment:
strftime is for formatting output of a datetime object, strptime is for parsing and the timezones always suck